dataformat-json-jackson/src/test/java/org/camunda/spin/json/tree/JsonTreeEditListPropertyTest.java

```diff
@@ -599,4 +599,40 @@ public void removeAtWithNegativeIndexOutOfBounds() {
       // expected
     }
   }
+
+  // ----------------- 10) contains ----------------------
```

Comment:
There is a javadoc comment on class level referencing these section comments. I think we should add number 10 there as well.